REFERENCE.md

```diff
@@ -0,0 +1,233 @@
+# Reference
+<!-- DO NOT EDIT: This document was generated by Puppet Strings -->
+
+## Classes
+
+* [`ibm_installation_manager`](#ibm_installation_manager): Init class for installing the IBM Installation Manager.
+
+## Resource types
+
+* [`ibm_pkg`](#ibm_pkg): Custom type for installing an IBM package.
+
+## Classes
+
+### ibm_installation_manager
+
+Init class for installing the IBM Installation Manager.
+
+#### Examples
+
+##### Installing Installation Manager to the default location of '/opt/IBM/InstallationManager'. In this example, we've downloaded and extracted the Installation Manager packages (installer) to '/vagrant/ibm/IM'
+
+```puppet
+class { 'ibm_installation_manager':
+  source_dir => '/vagrant/ibm/IM',
+}
+```
+
+##### Providing a zip source and a custom install location. This will retrieve the zip file from /mnt/IBM and extract it to /opt/IBM/tmp/InstallationManager. Installation Manager will then be installed to '/opt/myorg/InstallationManager'
+
+```puppet
+class { 'ibm_installation_manager':
+ source     => '/mnt/IBM/im.zip',
+ source_dir => '/opt/IBM/tmp/InstallationManager',
+ target     => '/opt/myorg/InstallationManager',
+}
+```
+
+#### Parameters
+
+The following parameters are available in the `ibm_installation_manager` class.
+
+##### `deploy_source`
+
+Data type: `Any`
+
+Specifies whether this module should be responsible for deploying the source package for Installation Manager. Valid values are `true` and `false`. Defaults to `false`
+
+Default value: `false`
+
+##### `source`
+
+Data type: `Any`
+
+**Required** if `deploy_source` is true. Specifies the source. This can be either an absolute path to the source or an HTTP address. This expects a compressed archive from IBM (zip).
+
+Default value: `undef`
+
+##### `source_dir`
+
+Data type: `Any`
+
+Specifies the absolute path to the directory to deploy the installer from. (This is the directory containing the `installc` binary.) If you extracted the archive yourself, point this parameter to the extracted archive. Defaults to `/opt/IBM/tmp`.
+
+Default value: `undef`
```

Comment:
Duplicate default values and they don't match.

Reply:
It's also wrong